### PR TITLE
Only allow admins to touch the course list

### PR DESCRIPTION
**Pull request by jamestjw**

### CODE REVIEW

File: public/course_list.php [hunk 1]
```diff
@@ -160,16 +167,16 @@
             </tr>
             <?php endforeach;?>
             <tr>
+                <td class="tgNorm"></td>
                 <td class="tgNorm"></td>
                 <form method="post">
                 <td class="tgNorm">
                     <select name = "course_selection" id="course_selection">
-                    <option value = "">--Select Course--</option>
+                    <option value = "" disabled seleted>--Select Course--</option>
```
Comment (chris7neves) [inline public/course_list.php:175]: seleted? Is that a typo?

File: public/course_list.php [hunk 2]
```diff
@@ -181,7 +188,7 @@
             <form method="post">
 
             <select name = "course_selection" id="course_selection">
-            <option value = "">--Select Course--</option>
+            <option value = "" disabled seleted>--Select Course--</option>
```
Comment (chris7neves) [inline public/course_list.php:191]: same here

**Review by chris7neves — CHANGES_REQUESTED** [review]

**Review by drouba — CHANGES_REQUESTED** [review]
The way you did it you only check once the form is submitted, but the form is still visible to anyone. I think the form should be visible only if you are an admin. 
Also eventually I think people who are not admins should only see the courses they are registered/linked to

File: public/course_list.php (hunk 1 above)
Comment (jamestjw) [reply public/course_list.php]: ok whoops resolved both

**Comment by jamestjw** [comment]
> The way you did it you only check once the form is submitted, but the form is still visible to anyone. I think the form should be visible only if you are an admin.
> Also eventually I think people who are not admins should only see the courses they are registered/linked to

Ok I made the forms hidden to non admins. as for people only seeing the courses that are relevant to them, Zack is supposed to be working on that

**Review by drouba — APPROVED** [review]
All good then!

**Pull request merged**